test/parallel/test-https-hwm.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+// Test https highWaterMark
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const https = require('https');
+const fixtures = require('../common/fixtures');
+
+const options = {
+  key: fixtures.readKey('agent1-key.pem'),
+  cert: fixtures.readKey('agent1-cert.pem')
+};
+
+let counter = 0;
+
+function loadCallback(highWaterMark) {
+  return function cb(res) {
+    assert.strictEqual(highWaterMark, res.readableHighWaterMark);
+    counter--;
+    console.log('back from https request. ',
+                `highWaterMark = ${res.readableHighWaterMark}`);
+    if (counter === 0) {
+      httpsServer.close();
+      console.log('ok');
+    }
+    res.resume();
+  };
+}
+
+function reqHandler(req, res) {
```

Comment:
unnecessary function, just inline this

Reply:
done

Reply:
@Himself65 Can you help to continue the review ? I need more approved.